extensions/adobe/experience/decisioning/proposition-response-details.schema.json

```diff
@@ -0,0 +1,65 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/response-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision Response Details",
+  "type": "object",
+  "meta:abstract": true,
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "A decision response is used to link an experience event that is a response to a prior decision/proposition events. Each decision output has a unique ID that can be echo'ed back with an experience event. The decision is assumed to have contributed to the experience event and is now attributed with by it.",
+  "definitions": {
+    "decision-response-datatype": {
+      "type": "object",
+      "properties": {
+        "xdm:decisionEventId": {
+          "type": "string",
+          "title": "Decision Event Identifier",
+          "description": "The unique identifier from the decision event. For a given decision request and profile the output of the decision is a unique event. If the same decision is made again for the same profile and the output are the same options, it is ist considered a different proposition with a unique propositionID. See https://ns.adobe.com/experience/decisioning/proposition."
```

Comment:
spelling issue - `it is ist`

Reply:
Corrected. Thanks @lrosenthol for spotting it.